Documentation/cli-installation-scenarios.md

```diff
@@ -242,8 +262,16 @@ OSX/Linux:
 
 #### Windows obtain one-liner example 
 
+CMD:
+
+```
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex}"
+```
+
+PowerShell:
+
 ```
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1'))}"
+iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex
```

Comment:
@MichaelSimons, now the script is not failing as it was last night. You are probably right that this might be the case when download fails and control went straight to `finally` block without setting `$Stream`. The easy fix would probably be:

```diff
 finally {
-    if ($HttpClient -ne $null) {
+    if((gv -name HttpClient -ErrorAction SilentlyContinue) -and $HttpClient -eq $null) {
+    # who said powershell isn't verbose enough! :P
         $HttpClient.Dispose()
     }
-    if ($Stream -ne $null) {
+    if((gv -name Stream -ErrorAction SilentlyContinue) -and $Stream -eq $null) {
         $Stream.Dispose()
     }
 }
```